services/actions/notifier_helper.go

```diff
@@ -103,6 +105,76 @@ func (input *notifyInput) Notify(ctx context.Context) {
 	}
 }
 
+func handleSchedules(
+	ctx context.Context,
+	schedules map[string][]byte,
+	commit *git.Commit,
+	input *notifyInput,
+) error {
+	if commit.Branch != input.Repo.DefaultBranch {
+		log.Trace("commit branch is not default branch in repo")
```

Comment:
We need to know which repo and which commit, otherwise this log is useless.

Reply:
But it's only a trace log for developing. In this case the commit is ignored intentionally, no need to tell end users or site admin. Putting repo into it or not doesn't have much difference (only slightly better ...)